src/I18n/I18n.php

```diff
@@ -246,6 +238,20 @@ public static function defaultLocale()
     }
 
     /**
+     * Set default fallback domain. If a domain can't provide a translation
+     * messages from the fallback domain are used.
+     *
+     * Get the currently set domain if argument is null.
+     *
+     * @param string|null $name name of the fallback-translator
+     * @return string
+     */
+    public static function defaultFallbackDomain($name = null)
```

Comment:
It seems that the fallback isn't exactly a favorite. Maybe setting the default domain should be dropped completely form the PR? Don't encourage future use?

As I said, my main goal was to disable it, not to change the domain.